node/server/server.ts

```diff
@@ -181,7 +183,7 @@ interface ServeOptions {
   distImportMapPath?: string
   featureFlags?: FeatureFlags
   inspectSettings?: InspectSettings
-  importMapPaths?: string[]
+  importMapPaths?: string[] | (() => Promise<string[]>)
```

Comment:
Can you explain when we might want `importMapPaths` to be a function? I don't see us calling it anywhere?

Reply:
Oh I think that's a remnant!

Reply:
removed in https://github.com/netlify/edge-bundler/pull/548/commits/556a32d4e0432217d9ab292d225a88e386533f29